x-pack/test/security_solution_cypress/cypress/tasks/api_calls/prebuilt_rules.ts

```diff
@@ -213,15 +228,10 @@ export const createAndInstallMockedPrebuiltRules = ({
   rules: Array<typeof SAMPLE_PREBUILT_RULE>;
   installToKibana?: boolean;
 }) => {
-  cy.log('Install prebuilt rules', rules?.length);
-  preventPrebuiltRulesPackageInstallation();
-  // TODO: use this bulk method once the issue with Cypress is fixed
-  // bulkCreateRuleAssets({ rules });
-  rules.forEach((rule) => {
-    createNewRuleAsset({ rule });
-  });
+  addAvailableToInstallPrebuiltRules(rules);
 
   if (installToKibana) {
+    cy.log('Install prebuilt rules', rules.length);
```

Comment:
Now that we have `addAvailableToInstallPrebuiltRules`, or however we want to call it, I think `createAndInstallMockedPrebuiltRules` should always install the rules that were created as assets, without having the optional `installToKibana` flag as param. So if you don't actually want to install the rules, you should just use `addAvailableToInstallPrebuiltRules`.

Anyways, that is maybe out of scope of this PR, we can follow up on it.

Reply:
You are right, this is what we want eventually but it's out of scope of this PR. I'll come up with a refactoring PR to get rid of `installToKibana` param.